agents/exec/src/main/resources/org.eclipse.che.exec.script.sh

```diff
@@ -9,6 +9,15 @@
 #   Codenvy, S.A. - initial API and implementation
 #
 
+
+is_current_user_root() {
+    test "$(id -u)" = 0 && return 0 || return 1
+}
+
+is_current_user_sudoer() {
+    sudo -n true >& /dev/null && return 0 || return 1
+}
```

Comment:
>& is bash
you should use
```
 is_current_user_sudoer() {
     sudo -n true > /dev/null 2>&1 && return 0 || return 1
 }
```


and maybe it could be simplified to
```
is_current_user_sudoer() {
  sudo -n true > /dev/null 2>&1
}
```